pkg/apis/crd/v1alpha1/types.go

```diff
@@ -600,3 +600,35 @@ type NamespacedName struct {
 	Name      string `json:"name,omitempty"`
 	Namespace string `json:"namespace,omitempty"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type ExternalNode struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec ExternalNodeSpec `json:"spec,omitempty"`
+}
+
+// ExternalNodeSpec defines the desired state for ExternalNode.
+type ExternalNodeSpec struct {
+	// Only one network interface is supported now, and we will only process interfaces[0].
```

Comment:
"and we will only process interface[0]" -> "Other interfaces except interfaces[0] will be ignored if there are more than one interfaces".

Reply:
Done.